docs/reference/commandline/system_df.md

```diff
@@ -0,0 +1,64 @@
+<!--[metadata]>
++++
+title = "system df"
+description = "The system df command description and usage"
+keywords = [system, data, usage, disk]
+[menu.main]
+parent = "smn_cli"
++++
+<![end-metadata]-->
+
+# system df
+
+```markdown
+Usage:	docker system df [OPTIONS]
+
+Show docker filesystem usage
+
+Options:
+      --help      Print usage
+  -v, --verbose   Show detailed information of space usage
+```
+
+The `docker system df` command displays information regarding the
+amount of disk space used by the docker daemon.
+
+By default the command will just show a summary of the data used:
+```bash
+$ docker system df
+TYPE                TOTAL               ACTIVE              SIZE                UNUSED
```

Comment:
`RECLAIMABLE` SGTM. I'm not sure it has to match the description of `-a, --all` though.